test/Network.test.js

```diff
@@ -12,18 +12,46 @@ var Validator = require("./../lib/shared/Validator").default;
 
 
 /**
+ * Merge all options of object b into object b
+ * @param {Object} a
+ * @param {Object} b
+ * @return {Object} a
+ *
+ * Adapted merge() in dotparser.js
+ */
+function merge (a, b) {
```

Comment:
I feel like this function is really unnecessary. Lodash and even es6 have functions to do this. I would way prefer including lodash than using this function.